docs/1.0-Installation.md

```diff
@@ -12,14 +12,11 @@ Quick Access:
     - [1.5.2 Cluster Mode](#152-cluster-mode)
 
 
-
 ## 1.1 - Prerequisites
 Please ensure you have completed all necessary prerequisite steps before moving onto the Workflow Section below.
 
 [All required prerequisites steps can be found in this section.](1.1-InstallationPrerequisites.md)
 
-Please utilise the [Glossary](0.1-Glossary.md) for any terms or acronyms found in this documentation which you may find unclear. 
```

Comment:
Is there a reason to remove this?

Reply:
Hm? Should slipped during rebase. Feel free to add it back.